code/__HELPERS/text.dm

```diff
@@ -1206,6 +1206,16 @@ GLOBAL_LIST_INIT(binary, list("0","1"))
 		else
 			. = ""
 
+/proc/weight_class_to_tooltip(w_class)
+	switch(w_class)
+		if(WEIGHT_CLASS_TINY to WEIGHT_CLASS_SMALL)
+			return "This item can fit into pockets, boxes and backpacks."
+		if(WEIGHT_CLASS_NORMAL)
+			return "This item can fit into backpacks."
+		if(WEIGHT_CLASS_BULKY to WEIGHT_CLASS_GIGANTIC)
+			return "This item is too large to fit into any storage."
```

Comment:
```suggestion
			return "This item is too large to fit into any standard storage."
```
There's a few bulky items which *can* fit into storage, but they can only fit in specialized storage (sabre -> sheathe), so this is mildly misleading